.github/workflows/pull-request.yml

```diff
@@ -49,8 +49,8 @@ jobs:
       - name: Install Dependencies
         run: npm ci
 
-      - name: Run ESLint to check for linting errors
-        run: npm run lint:check
+      - name: Run ESLint to check for linting errors in modified files
+        run: npx eslint $(git diff --name-only --diff-filter=d -- '***.ts' ${{ github.event.before }} ${{ github.sha }})
```

Comment:
Change `npx` to `npm`

Reply:
It can't be changed here, as I need the github context in this command, which is not available if I run the script via npm run.
Is there any particular reason for not using npx?

Reply:
Aight. Not really, just a preference and convention followed